packages/product-json-to-csv/package.json

```diff
@@ -47,6 +47,7 @@
     "highland": "2.13.0",
     "json2csv": "3.11.5",
     "lodash": "4.17.10",
+    "lodash.isnil": "^4.0.0",
```

Comment:
No need to install this dependency since the whole `lodash` package is installed above on line 49.

Reply:
for the future; be careful to pin the version that you install

Reply:
Removed.
👍